app/models/provider.rb

```diff
@@ -63,4 +64,10 @@ def address_info
       .attributes_before_type_cast
       .slice('address1', 'address2', 'address3', 'address4', 'postcode', 'region_code')
   end
+
+  def update_changed_at(timestamp: Time.now.utc)
+    # We don't want `updated_at` to change, only `changed_at`. Otherwise what
```

Comment:
so what's the definition?

When something is changed it's not an update?

Reply:
`updated_at` - when the current object has been updated
`changed_at` - when anything on this, or related, object(s) have changed which will necessitate re-publishing to the apply system (via the incremental load system)